src/clj_kondo/impl/linters.clj

```diff
@@ -484,9 +484,10 @@
                 ctx)
           ctx (assoc ctx :lang (:lang ns))]
       (when-not (identical? :off (-> ctx :config :linters :used-underscored-binding :level))
-        (doseq [binding (set (->> (:used-bindings ns)
-                                  (filter (comp not :clj-kondo.impl/generated))
-                                  (filter #(str/starts-with? (str (:name %)) "_"))))]
+        (doseq [binding (into #{}
```

Comment:
Just realized, we don't need a set at all, we can add `distinct` in the transducer.

Reply:
too late now  ;)

Reply:
but given the fact that distinct internally uses a set (even if it's volatile), it's not super surprising. but i realized i only rarely use transducers so dusting that off wasn't too bad :)